R/si_gravity.R

```diff
@@ -0,0 +1,27 @@
+si_gravity <- function(O_i, D_j, C_ij, flows, data,
+                       mu = 1, alpha = 1, beta = -1){
+  
+  data = data %>% 
```

Comment:
Two minor issues with this:

1. Use of both equals and arrow assignment in same script file, please use `=` throughout as that's the package style
2. Use of pipe operator in function calls, not sure about this but if it works I'm happy to give it a spin